h/activity/query.py

```diff
@@ -42,7 +43,8 @@ def extract(request, parse=parser.parse):
     If no query is present in the passed request, returns ``None``.
     """
 
-    q = parse(request.params.get('q', ''))
+    schema = SearchParamsSchema()
```

Comment:
Lyza asked why: Because this function takes in a request obj and pulls stuff off the request obj besides the params I think it's easier to leave this function's api as is and just put the schema call inside here. The alternative is breaking out all the things this function needs off the request obj and passing them in as separate args.